devops/openshift/database/patroni-deploy.yml

```diff
@@ -31,10 +31,6 @@ objects:
       role: master
       cluster-name: ${NAME}
       app.kubernetes.io/name: patroni
-      sessionAffinity: None
```

Comment:
Is those properties the cause the problem to not work and need to do some commands for this to have a main instance?

Reply:
So basically removing these properties allows for connection with services outside this cluster?